### PR TITLE
chore(samples): add create start instance samples

### PR DESCRIPTION
**Pull request by FrodoTheTrue**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-compute/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by SurferJeffAtGoogle** [comment]
A test failed:
```
 1) create start instance tests
       should create instance with subnet:
     Error: Command failed: node instances/create-start-instance/createInstanceWithSubnet long-door-651 europe-central2-b gcloud-test-instance-d378d517 global/networks/default regions/europe-central2/subnetworks/default
Invalid value for field 'resource.networkInterfaces[0].subnetwork': 'regions/europe-central2/subnetworks/default'. The referenced subnetwork resource cannot be found.

      at checkExecSyncError (child_process.js:635:11)
      at Object.execSync (child_process.js:671:15)
      at execSync (test/createStartInstance.test.js:29:28)
      at Context.<anonymous> (test/createStartInstance.test.js:208:20)
```

**Comment by FrodoTheTrue** [comment]
@SurferJeffAtGoogle it looks like the project doesn't have default subnetworks.

File: samples/instances/create-start-instance/createInstanceWithAdditionalDisk.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        machineType: `zones/${zone}/machineTypes/n1-standard-1`,
+        networkInterfaces: [
+          {
+            name: 'global/networks/default',
+          },
+        ],
+      },
+    });
+    let operation = response.latestResponse;
+    const operationsClient = new compute.ZoneOperationsClient();
```
Comment (bcoe) [inline samples/instances/create-start-instance/createInstanceWithAdditionalDisk.js:79]: I'm surprised this doesn't return an awaitable operation and that you need to poll. Is this something we'll improve on in the future (I think the non REST gapics you can just await long running operations?).

File: samples/test/createStartInstance.test.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    const output = execSync(
+      `node instances/create-start-instance/createInstanceWithAdditionalDisk ${projectId} ${zone} ${instanceName}`
+    );
+    assert.match(output, /Instance created./);
+
+    execSync(`node deleteInstance ${projectId} ${zone} ${instanceName}`);
+  });
+
+  it('should create instance with subnet', async () => {
+    const projectId = await instancesClient.getProjectId();
```
Comment (bcoe) [inline samples/test/createStartInstance.test.js:206]: The problem with performing the delete step as part of a test, is that integration tests like this tend to be somewhat flaky (and in these edge cases we'll leak compute instances).

A pattern we've used for other long lived resources is to use a well known prefix for the instance we're creating, e.g., `test-instance-abc123`, and to then periodically delete any instances older than a given number of hours. Here's an example:

https://github.com/googleapis/nodejs-bigtable/blob/80022f846081a67c610870cec0e26f1da76d0dd5/samples/test/util.js#L36

**Review by bcoe — CHANGES_REQUESTED** [review]
I think it would be worth adding a cleanup step that ensures we've removed stale instances, if we don't have something similar already.

File: samples/instances/create-start-instance/createInstanceWithAdditionalDisk.js (hunk 1 above)
Comment (FrodoTheTrue) [reply samples/instances/create-start-instance/createInstanceWithAdditionalDisk.js:79]: Yep, support for LRO will be released soon

**Comment by bcoe** [comment]
@FrodoTheTrue there's still a failure:

```
     Error: Command failed: node instances/create-start-instance/createInstanceWithSubnet long-door-651 europe-central2-b gcloud-test-instance-851aafd2 global/networks/default-compute regions/us-central1/subnetworks/default
```

I believe that the subnetwork should be `default-compute` too.

**Comment by FrodoTheTrue** [comment]
@bcoe Yep, have just updated subnetwork too.
Also, I have created separate bug for deleting test instances: b/215012704.

I am going to do separate PR for this and support deleting instances for all test samples (not only for create/start).
WDYT?

**Review by bcoe — APPROVED** [review]
